llama/marc_export.py

```diff
@@ -0,0 +1,73 @@
+import os
+from datetime import datetime
+
+import boto3
+from s3_concat import S3Concat
+
+
+def get_file_keys_from_bucket(client, bucket, key_prefix, file_type=""):
+    """Get file keys from bucket that match with key prefix and file type."""
+    bucket_file_list = client.list_objects(Bucket=bucket)["Contents"]
+    file_keys = []
+    for file in [
+        f
+        for f in bucket_file_list
+        if f["Key"].endswith(file_type) and f["Key"].startswith(key_prefix)
+    ]:
+        file_keys.append(file["Key"])
+    return file_keys
+
+
+def move_s3_file(client, old_bucket, old_key, new_bucket, new_key):
+    """Duplicate behavior of AWS CLI mv command by copying the source object to a new key
+    and then deleting the source object."""
+    client.copy_object(
+        Bucket=new_bucket,
+        CopySource=f"{old_bucket}/{old_key}",
+        Key=new_key,
+    )
+    client.delete_object(
+        Bucket=old_bucket,
+        Key=old_key,
+    )
+
+
+def marc_update_export(key_prefix):
```

Comment:
Depending on how much is left after the changes below are made, it's possible it will make more sense for most or all of this logic to just live in the `cli` comand.